### PR TITLE
Implemented output with strategy pattern

### PR DESCRIPTION
**Pull request by eduard-sukharev**
Added a DumpInterface that defines `dump($path, $contents)` method for dumping the generated result.

### CODE REVIEW

File: src/Opensoft/RadApiTool/ConsoleDump.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+     * @param $path
+     * @param $content
+     * @return mixed
+     */
+    public function dump($path, $content)
+    {
+        $this->output->writeln(sprintf('File: "%s":', $path));
+        $this->output->writeln($content);
+    }
+}
```
Comment (fightmaster) [inline src/Opensoft/RadApiTool/ConsoleDump.php]: cs

File: src/Opensoft/RadApiTool/DumpInterface.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+/**
+ * Interface DumpInterface
+ *
+ * @package Opensoft\RadApiTool
+ * @author Eduard Sukharev <eduard.sukharev@opensoftdev.ru>
+ */
+Interface DumpInterface
+{
+    /**
+     * @param $path
```
Comment (fightmaster) [inline src/Opensoft/RadApiTool/DumpInterface.php]: cs

File: src/Opensoft/RadApiTool/Command/DtoGeneratorCommand.php [hunk 3]
```diff
@@ -25,6 +28,11 @@
  */
 class DtoGeneratorCommand extends Command
 {
+    /**
+     * @var DumpInterface
+     */
+    private $resultOutput;
```
Comment (fightmaster) [inline src/Opensoft/RadApiTool/Command/DtoGeneratorCommand.php]: is it needed?

Comment (eduard-sukharev) [inline src/Opensoft/RadApiTool/Command/DtoGeneratorCommand.php]: i guess you're right - i can do without it. it's initialized and only used in `execute()`, so we can drop it.

**Pull request merged**